recipe/conda_build_config.yaml

```diff
@@ -32,16 +33,18 @@ runtime_version:
 # reported in the VS help->about UI.  It is perhaps a more readily
 # referenceable number.
 update_version:
- - 9
- - 9
+ - 10
+ - 10
  - 8
  - 8
```

Comment:
Can you keep `9` and `10` instead of `8` and `10`?

Reply:
Done